hack/.golint_failures

```diff
@@ -751,6 +751,8 @@ test/e2e/scalability
 test/e2e/scheduling
 test/e2e/servicecatalog
 test/e2e/storage
+test/e2e/storage/drivers
```

Comment:
Do you mind fixing up the lint errors, so we don't increase the technical debt? please.

Reply:
Thank you for your comment.
Actually, it is added to avoid "should not use dot imports" errors caused by dot importing ginkgo and gomega. I will fix up lint errors other than that.